### PR TITLE
Weekly announcement for week 46 (Nov 13, 2020)

### PR DESCRIPTION
**Pull request by cokiengchiara**
Check for:
- [x] Managed Apps https://github.com/giantswarm/roadmap/issues/227 @giantswarm/team-halo 
- [x] Web UI @giantswarm/sig-ux 
- [x] Docs ([Changes](https://github.com/giantswarm/docs/commits/master) from Nov 2) @giantswarm/sig-docs 

Since I will be out on Friday, some docs:
* Reference on making [provider-independent announcements](https://intranet.giantswarm.io/docs/product/releases/coordination/#process-drafting-batched-announcement) 
* opsctl release notify [cheat sheet](https://intranet.giantswarm.io/docs/product/releases/opsctl-notify-release-cheat-sheet/)

Documentation changes:
- [x] @alex-dabija 'Automatic termination of unhealthy nodes' https://github.com/giantswarm/docs/commit/ffde1a511095474e55c802faee39c719561cda10

### CODE REVIEW

File: announcements/2020-Week-46/announcement.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# :zap: This Week in Provider-Independent Announcements (November 13, 2020):zap:
+
+## Giant Swarm Catalog (Managed Apps)
+
+1. [Cert Manager v2.3.2](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#232---2020-11-09) upgrades to upstream v1.0.4 and adds small improvements.
```
Comment (cokiengchiara) [inline announcements/2020-Week-46/announcement.md:5]: @glitchcrab Are the other changes noteworthy?

Comment (glitchcrab) [reply announcements/2020-Week-46/announcement.md:5]: It does add the values schema so it may well be worth mentioning that.

Comment (cokiengchiara) [reply announcements/2020-Week-46/announcement.md:5]: lol i made that change of course i didn't think it's important hahaha

Comment (cokiengchiara) [inline announcements/2020-Week-46/announcement.md:5]:
```suggestion
1. [Cert Manager v2.3.2](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#232---2020-11-09) upgrades to upstream v1.0.4 and adds schema for validating user values.
```

@glitchcrab wdyt?

Comment (glitchcrab) [reply announcements/2020-Week-46/announcement.md:5]: Yup works for me, I like that.

**Comment by cokiengchiara** [comment]
thanks @alex-dabija for volunteering to take care this week.

**Comment by alex-dabija** [comment]
@giantswarm/team-halo, @giantswarm/sig-ux, @giantswarm/sig-docs Could you please update this PR with any relevant changes for our customers? Thank you!

**Comment by alex-dabija** [comment]
@marians Could you review the documentation section of the announcement?

@glitchcrab Could you review the managed apps section of the announcement?

**Comment by alex-dabija** [comment]
I publish the draft announcement to #sig-product.

**Comment by alex-dabija** [comment]
I published the announcement to all customers.

**Comment by marians** [comment]
Any reason why this PR is still open?

**Comment by alex-dabija** [comment]
Nobody approved it yet. I was going to just merge it.

**Pull request merged**